### DISCUSSION TITLE
Bots take batteries out of the "Handle with extreme care" cargo

### DISCUSSION THREAD

**Discussion by Kokhirus** [FakeFishGames/Barotrauma#14031] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If your submarine has spare space inside its batteries, bots will probably take newly placed batteries from explosive cargo and let you get blown up.

### Reproduction steps

1. Empty submarine's batteries
2. Tell a bot to recharge batteries
3. Start a "Handle with extreme care" mission
4. Switch the crates' default batteries to your own
5. Wait for the bot
6. Get blown up

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/b4c7ddbf26d59f08544ebc9122f7a9460676d51e (commit in our private repo)

**Comment by Jasontti** [comment]
Tested the fix and found no issues.
Closing discussion as resolved.
